### ISSUE TITLE
Validation Loss is `inf`

### ISSUE THREAD

**Issue by jmiller656** [GuitarML/GuitarLSTM#10]
I've run into issues where the validation loss is `inf` during training. I think there might be an issue with the loss function

**Comment by GuitarML** [comment]
This has happened to me when there are 0’s in the audio data, because the loss calculation would divide by 0. A more robust loss calculation would be a good idea. Thanks for bringing this up!
